src/Symfony/Bundle/SecurityBundle/DataCollector/SecurityDataCollector.php

```diff
@@ -309,6 +333,8 @@ public function getLogoutUrl()
      * Returns the FQCN of the security voters enabled in the application.
      *
      * @return string[]
+     *
+     * @deprecated Use the data returned by {@link getVoterDetails()} instead
      */
     public function getVoters()
     {
```

Comment:
do we use it still? Should we trigger a hard deprecation using trigger_error()?

Reply:
Right ! I added it, thanks :)